packages/static-wado-creator/lib/operation/ThumbnailService.js

```diff
@@ -110,10 +110,6 @@ class ThumbnailService {
   async generateThumbnails(itemId, dataSet, metadata, callback, options) {
     const { imageFrame, id } = this.favoriteThumbnailObj;
 
-    if (!options.thumb) {
```

Comment:
The thumbnail generation also generates the rendered video, so have to move the skip checks to after the video genreation.